### DISCUSSION TITLE
Can we use external app without Saleor cloud

### DISCUSSION THREAD

**Discussion by zeing** [saleor/apps#440] [Q&A] [answered]
Hi , I've created Saleor Core locally and would like to install local app in Dashboard

I got an error like this in Admin Dashboard
![image](https://user-images.githubusercontent.com/20161393/235709917-082b2751-137a-4ab4-b6a2-37181e5a1ecd.png)

dashboard v3.12.3
core v3.13.0

Can we use an external app without the Saleor cloud?

**Comment by lkostrowski** [comment, marked as answer]
Hey

This problem is not related to Saleor Cloud. 
Apps that are hosted by Saleor officially (with `.saleor.app` domain) can be installed only with Saleor Cloud. But all of them are open source and can be self hosted to be used with self hosted Saleor

Likely you have problem with your Saleor configuration. The error you see means that Saleor (during registration) gave invalid config to the app (token, API URL). For self hosted Saleor it can be `localhost` instead publically available address

Please check if this comment works for you https://github.com/saleor/apps/discussions/214#discussioncomment-5511776
